### DISCUSSION TITLE
How do i get usable funds and help needed to properly apply the full putential of an investment.

### DISCUSSION THREAD

**Discussion by Parzavil3** [community/community#30091] [Other Feature Feedback, Questions, & Ideas]
and help needed to properly apply the full putential of an investment

**Comment by Parzavil3** [comment]
I'm a chromium creator my real name is jurgens van der merwe I no longer have an I'd book. I don't have a car to get one or places to sort out the misunderstanding of my identity and the theft of it. I just want to fix it I've tried it all. It once I get funds I will embark on a committed journey to work and live in researching enhancing adding growing and seeing what is possible

**Comment by curlyjhc00** [comment]
https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
